### ISSUE TITLE
Consider merging with org-project-capture (formerly org-projectile)

### ISSUE THREAD

**Issue by colonelpanic8** [delehef/org-project#3]
I've generalized org-projectile to support a project backend in this branch:

https://github.com/colonelpanic8/org-projectile/tree/org-project-capture

Which im likely to release very soon.

I'm pretty sure that the project backend that it provides will support all of the features of this package and more with a very similar interface.

I think it would be better for the community if the code supporting projectile and project were in the same place like this, and I think it might make sense to have org-project simply be a very very thin layer that points at what will be org-project-capture.

I stole your trick of using a dummy capture template to make the org-capture stuff work nicely btw. Not sure why I didn't think of that, but nice job!

Anyway, let me know what you think.

Would also love help testing!

**Comment by colonelpanic8** [comment]
I've also noticed that you haven't uploaded to melpa yet.

Would you mind if I uploaded a project called org-project to melpa?

**Comment by delehef** [comment]
Hi!

If the new update does not drag projectile in as a dependency, I will probably simply close this repo, as working with vanilla project.el and no dependency on projectile was its raison d'être.

> Would you mind if I uploaded a project called org-project to melpa?

No, your project is more maintained than mine, please feel free to take the name on MELPA.

**Comment by colonelpanic8** [comment]
> If the new update does not drag projectile in as a dependenc

yes, org-project-capture has no projectile dependency, and uses the project backend by default.
